azure-devops/global/abstracts/run-test-e2e.yaml

```diff
@@ -16,32 +16,35 @@ steps:
 
   - template: ./replace-environment-variables.yaml
     parameters:
-      FILE: '$(Agent.TempDirectory)/OpenSearch-Dashboards/plugins/${{ parameters.PLUGIN_NAME }}/.ci/config/opensearch_dashboards.yml'
-      DISPLAY_NAME: 'Replace Environment Variables'
+      FILE: "$(Agent.TempDirectory)/OpenSearch-Dashboards/plugins/${{ parameters.PLUGIN_NAME }}/.ci/config/opensearch_dashboards.yml"
```

Comment:
When you have variables in the string, you are right, should be double quotes. But if it's just string not variables, should be single quotes.